developerportal/templatetags/app_filters.py

```diff
@@ -73,12 +73,15 @@ def split_across_two_columns(list_):
 
 @register.filter
 def domain_from_url(url, request):
+    output = ""
     parsed = urlparse(url)
     if parsed.netloc:
         try:
             output = ".".join(parsed.netloc.split(".")[-2:])
         except Exception:  # Deliberately broad
```

Comment:
I know this is from a previous commit but I'd be curious why it's "Deliberately broad" and why it can't be narrowed

Reply:
True, it could be narrowed to `AttributeError` and  `IndexError` looking at it.  Will do that before merging.

I think the original intent was to allow nothing to blow up the rendering of a page when the content of the page can't be guaranteed to be legit